### PR TITLE
RELATED: RAIL-4669 Web components docs improvements

### PR DESCRIPTION
**Pull request by andriichumak**
* A new article on Angular integration
* A new article on Vue integration
* Added numerous warnings about setup prerequisites, not supported plugins etc.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1218/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/815/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1219/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/816/ : SUCCESS in 8s

File: docs/19_webcomponents_intro.md [hunk 1]
```diff
@@ -75,13 +67,13 @@ section of your web page.
 The script **must be** of the type `module`, as we are using JavaScript modules for this distribution.
 
 The library will parse its own URL to pre-configure and allow you to skip the boilerplate code:
-* The domain name `{your-gd-server-url}` must be the domain of your GoodData.CN server or the GoodData Cloud instance. 
-    This is the domain where the script will be loaded from as well as the domain that will be used to load your insight and dashboard data. You cannot load the script from one instance to use it with data from another instance.
-    **At the moment it's not possible to connect to several GoodData instances from a single runtime.**
+* The domain name `{your-gd-server-url}` must be the domain of your GoodData.CN server or the GoodData Cloud instance.
```
Comment (no23reason) [inline docs/19_webcomponents_intro.md:70]: We always mention GoodData Cloud first, GoodData.CN second

File: docs/19_webcomponents_intro.md [hunk 2]
```diff
@@ -91,10 +83,63 @@ on the page:
 * `<gd-dashboard />` for [dashboard embedding][6].
 * `<gd-insight />` for [insight embedding][7].
 
+## Prerequisites and limitations
+
+### Supported web browsers
+
+Since Web Components is a relatively new technology, the library will not work in older browsers, such as
+**Internet Explorer**. For details, refer to the
+<a href="https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry#browser_compatibility" target="_blank">Custom Elements</a> browser compatibility sections on MDN.
```
Comment (no23reason) [inline docs/19_webcomponents_intro.md:92]: Nitpick: please add `rel="noopener noreferrer"`

File: docs/19_webcomponents_intro.md [hunk 3]
```diff
@@ -91,10 +83,63 @@ on the page:
...
+
+### Cross-Origin Resource Sharing (CORS) configuration 
+
+You will also need to set up a **CORS configuration** on the GoodData server instance to allow the script from your application
+domain to make network requests to the GoodData server. Refer to the CORS configuration sections in [GoodData.CN][3] and
+[GoodData Cloud][4] documentation.
+
+### Content Security Policy (CSP) configuration
+
+You might need to adjust the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP" target="_blank">CSP</a> headers of your server, if you are using this technology.
```
Comment (no23reason) [inline docs/19_webcomponents_intro.md:102]: Nitpick: please add `rel="noopener noreferrer"` to the `a` tag.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1220/ : SUCCESS in 2m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/817/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1221/ : SUCCESS in 1m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/818/ : SUCCESS in 6s

**Comment by andriichumak** [comment]
Hey @no23reason. Updated according to your comments and backported changes to the 8.11 version. If you're OK with the content, let's merge and deploy. Thanks!

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR574

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1223/ : SUCCESS in 1m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/170/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/365/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul/7/ : FAILURE in 3m 01s (non-voting)

**Pull request merged**